File: Algorithms/Implementation Challenges/Breaking the Records/BreakingtheRecords.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int highest = 0, lowest = 0;
        int highBreak = 0, lowBreak = 0;
        for(int i=0; i < n; i++){
            int record = in.nextInt();
            if (i == 0) {
                highest = record;
                lowest = record;
            } else {
                if (record > highest) {
                    highest = record;
                    highBreak++;
                } else if (record < lowest) {
                    lowest = record;
                    lowBreak++;
                }
            }
        }
        System.out.printf("%d %d", highBreak, lowBreak);
    }
}

File: Algorithms/Implementation Challenges/Breaking the Records/BreakingtheRecords.py
#!/bin/python3

import sys


n = int(input().strip())
s = list(map(int, input().strip().split(' ')))
highest = s[0]
lowest = s[0]
highBreak = 0
lowBreak = 0
for i in range(1, n):
    if s[i] > highest:
        highest = s[i]
        highBreak += 1
    elif s[i] < lowest:
        lowest = s[i]
        lowBreak += 1
print (highBreak, lowBreak)
